pkg/plugins/backendplugin/pluginextensionv2/rendererv2.proto

```diff
@@ -18,14 +18,28 @@ message RenderRequest {
   int32 timeout = 8;
   string timezone = 9;
   map<string, StringList> headers = 10;
-  string renderType = 11;
 }
 
 message RenderResponse {
   string error = 1;
+}
+
+message RenderCSVRequest {
+  string url = 1;
+  string filePath = 5;
```

Comment:
Why is it necessary to skip fields 2-4?

Reply:
I was too fast on that, thanks for the catch! 🤦